src/drivers/uavcan/uavcan_params.c

```diff
@@ -193,3 +193,803 @@ PARAM_DEFINE_INT32(UAVCAN_LGT_NAV, 3);
  * @group UAVCAN
  */
 PARAM_DEFINE_INT32(UAVCAN_LGT_LAND, 0);
+
+
+
+/**
+ * Actuator function assigned to CAN1
+ * See output_control FUNCTION values
+ *
+ * @reboot_required false
+ * @min 0
+ * @max 32768
+ * @group PWM
+ * @value 0 NONE
+ */
+PARAM_DEFINE_INT32(CAN_ACT_FUNC1, 0);
```

Comment:
UAVCAN prefix?

Reply:
FN also works to abbreviate function.

Reply:
Was just trying to come up with a prefix that made it clear it was CAN actuation, and fit within the 16-char param name limit for all suffixes. "FN" would help that

Reply:
I'm planning for the coexistence of more than just UAVCAN.